docs/roadmap/index.md

```diff
@@ -16,70 +18,70 @@ and [FluxCloud](https://github.com/justinbarrick/fluxcloud) notifications.
 
 Goals
 
-- Offer an in-place migration tool for those that are using Flux in read-only mode to synchronize plain manifests
-- Offer a migration guide for those that are using Flux in read-only mode to synchronize Kustomize overlays
-- ~~Offer a dedicated component for forwarding events to external messaging platforms~~
+- [ ]  Offer an in-place migration tool for those that are using Flux in read-only mode to synchronize plain manifests
+- [ ]  Offer a migration guide for those that are using Flux in read-only mode to synchronize Kustomize overlays
+- [x]  <span style="color:grey">Offer a dedicated component for forwarding events to external messaging platforms</span>
 
 Non-Goals
 
-- Migrate users that are using Flux to run custom scripts with `flux.yaml`
-- Automate the migration of `flux.yaml` kustomize users
+- [ ]  Migrate users that are using Flux to run custom scripts with `flux.yaml`
```

Comment:
Shouldn't this be a normal list? Given these are non-goals and are thus never going to be fulfilled.

Reply:
I think only the tasks should have checklists

Reply:
Sounds good!
How about this then?
<img width="1440" alt="Screenshot 2020-07-30 at 12 07 32" src="https://user-images.githubusercontent.com/92439/88910789-40d95d80-d25d-11ea-800e-ace64557c18f.png">

Reply:
Slightly triggered by the misalignment of the second line of text next to the :heavy_check_mark:, other than that :ok_hand: